### ISSUE TITLE
Add support for Mac icons

### ISSUE THREAD

**Issue by JakeSc** [KrauseFx/fastlane-plugin-appicon#15]
I was able to add rough support for Mac icon sizes by modifying `appicon/actions/appicon_action.rb` to include the Mac dimensions, like this:
```
      def self.needed_icons
        {
          iphone: {
            '2x' => ['20x20', '29x29', '40x40', '60x60'],
            '3x' => ['20x20', '29x29', '40x40', '60x60']
          },
          ipad: {
            '1x' => ['20x20', '29x29', '40x40', '76x76'],
            '2x' => ['20x20', '29x29', '40x40', '76x76', '83.5x83.5']
          },
          mac: {
            '1x' => ['16x16', '32x32', '128x128', '256x256', '512x512'],
            '2x' => ['16x16', '32x32', '128x128', '256x256', '512x512']
          },
        }
      end
```

This worked for my purposes. I would submit a PR, however this isn't completely correct. Because of the double array loop where the resizing occurs, the script also writes out the `iphone` and `ipad` sizes, resulting in some `Unassigned` / `unknown-trait-combination` entries in the resulting xcassets file. A better solution would be to not export he `iphone` and `ipad` sizes for the `mac` device. I don't know enough about how necessary these are for the other devices to make the change myself.

**Comment by JakeSc** [comment]
![image](https://user-images.githubusercontent.com/275063/27523602-129b3718-59e3-11e7-8dd6-0fdb1fd501c1.png)

**Comment by felipeplets** [comment]
@KrauseFx is there a standard way in a Fastlane Plugin to check which is the current platform?

**Comment by KrauseFx** [comment]
Yeah,`FastlaneCore::Helper.project.mac?` or `FastlaneCore::Helper.project.mac_app?`
